### PR TITLE
Better map performance using marker clustering

### LINKED ISSUE #132: Map is too slow
The map should be very easy to move around and zoom in and out. Please fix the performance issues of the map.

### PR DESCRIPTION
**Pull request by bencpeters**
This PR improves map performance by clustering markers at higher zoom levels to avoid poor redraw performance.

image

It also imports the SVG logo differently to allow for slightly better rendering performance.

The clustering is disabled by default at zoom level 6 & below (which is the default "flyTo" level):

image

**Note**
Right now, this branch is based off of the branch in PR #152, to allow for easy review of these changes. #152 should be merged before this, then I'll rebase & force-push and change the base to `master`.

Fixes #132

### CODE REVIEW

File: packages/website/src/routes/Homepage/Map/Markers/ColoredBuoyIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import L from "leaflet";
```
Comment (ericboucher) [inline packages/website/src/routes/Homepage/Map/Markers/ColoredBuoyIcon.tsx:1]: Can we delete this file?

File: packages/website/src/routes/Homepage/Map/Markers/buoy.svg [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ericboucher) [inline packages/website/src/routes/Homepage/Map/Markers/buoy.svg:1]: Maybe minify

**Review by ericboucher — APPROVED** [review]

**Comment by ericboucher** [comment]
Works great!

File: packages/website/src/routes/Homepage/Map/Markers/ColoredBuoyIcon.tsx (hunk 1 above)
Comment (bencpeters) [reply packages/website/src/routes/Homepage/Map/Markers/ColoredBuoyIcon.tsx:1]: Yes! Meant to...

**Pull request merged**